src/Ouroboros-Mini_Protocols.thy

```diff
@@ -399,7 +399,7 @@ text \<open>
 method state_machine_bisimulation uses program_expansion extra_splits = (
   subst (2) program_expansion,
   fastforce
-    simp add: protocol_state_machine.possibilities_def domIff
+    simp add: protocol_state_machine.possibilities_def Let_def domIff
```

Comment:
I think it’s better to have `Let_def` come after `domIff`. `domIff` is crucial (that’s related to how the conformance relation is defined), while `Let_def` is only needed if the user chooses to use `let`.